<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Decoder</class>
 <widget class="QMainWindow" name="Decoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Decoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QVBoxLayout" name="input_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="input_channels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>13</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="input_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>226</width>
            <height>327</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="input_list_layout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="input_chart" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="effect">
      <property name="title">
       <string>Effect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="output">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="output_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="output_channels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>13</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="output_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>226</width>
            <height>327</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="output_list_layout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="output_chart" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_file_load"/>
    <addaction name="menu_file_encode"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="channels_fs_0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8000 Hz</string>
   </property>
  </action>
  <action name="channels_fs_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>11025 Hz</string>
   </property>
  </action>
  <action name="channels_fs_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>22050 Hz</string>
   </property>
  </action>
  <action name="channels_fs_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>32000 Hz</string>
   </property>
  </action>
  <action name="channels_fs_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>44100 Hz</string>
   </property>
  </action>
  <action name="menu_file_load">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="menu_file_encode">
   <property name="text">
    <string>Encode new</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
